### PR TITLE
Votes dashboard

### LINKED ISSUE #314: Admin view to see casted vote
`Admin/Conference` moderator should be allowed to see vote casted by all reviewers in `/vote/`. Only admin should be allowed to see the content. Proposals should be grouped by section

### Filters for admin
screen shot 2015-07-19 at 10 18 20 pm
screen shot 2015-07-19 at 10 40 42 pm

### Proposal item
screen shot 2015-07-19 at 10 33 54 pm

Proposal item in this view should display `total vote count`, `each reviewer vote value, their comment`. Clicking on the `Accept/Reject proposal` should take to `http://<domain>/<conference>/proposals/test-1233454/review/` (rename the `/review` as `/change-proposal-review-state`. This should be only allowed for `admin/conference moderator`.

If the section reviewer hasn't voted, display their name and no vote against the name.

### PR DESCRIPTION
**Pull request by ChillarAnand**
This view might need some UI enhancements.

closes #314 #325 

Screenshots:

### CODE REVIEW

**Comment by kracekumar** [comment]
@ChillarAnand Screenshots are missing :-)

File: junction/proposals/forms.py [hunk 1]
```diff
@@ -166,3 +172,23 @@ def __init__(self, conference, *args, **kwargs):
         self.fields['proposal_section'].choices = _get_proposal_section_choices(conference)
         self.fields['proposal_type'].choices = _get_proposal_type_choices(conference)
         self.fields['reviewer_comment'].choices = ProposalReviewerComment.CHOICES
+
+
+class ProposalVotesFilterForm(forms.Form):
```
Comment (kracekumar) [inline junction/proposals/forms.py]: `ProposalVotesFilterForm` and `ProposalsToReviewForm` have some common fields. Can we reuse it ?

File: junction/proposals/urls.py [hunk 2]
```diff
@@ -26,7 +26,7 @@
     url(r'^(?P<slug>[\w-]+)/delete/$', views.delete_proposal, name='proposal-delete'),
     url(r'^(?P<slug>[\w-]+)/update/$', views.update_proposal, name='proposal-update'),
     url(r'^(?P<slug>[\w-]+)/upload-content/$', views.proposal_upload_content, name='proposal-upload-content'),
-    # url(r'^(?P<slug>[\w-]+)/review/$', views.review_proposal, name='proposal-review'),
+    url(r'^(?P<slug>[\w-]+)/change-proposal-review-state/$', views.review_proposal, name='proposal-review'),
```
Comment (kracekumar) [inline junction/proposals/urls.py]: `review_proposal` view should be accessible to only `super user or staff`. You need to add the check on the view, as of now `reviewers` can change it.

File: junction/proposals/dashboard.py [hunk 3]
```diff
@@ -164,3 +164,70 @@ def reviewer_comments_dashboard(request, conference_slug):
...
+
+    if request.method == 'GET':
+        for section in proposal_sections:
+            section_proposals = [p for p in proposals_qs if p.proposal_section == section]
+            proposals.append(s_items(section, section_proposals))
+
+        return render(request, 'proposals/votes-dashboard.html',
+                      {'conference': conference,
+                       'proposals': proposals,
+                       'form': form, })
```
Comment (kracekumar) [inline junction/proposals/dashboard.py]: Remove the space after `,` or remove comma and space.

**Comment by ChillarAnand** [comment]
Image 1:



Image 2:



Image 3:

**Pull request merged**